### DISCUSSION TITLE
VAE models disappeared from list?

### DISCUSSION THREAD

**Discussion by thestorytree** [vladmandic/sdnext#2989] [Q&A]
Since a few days ago, no longer can see a list of VAE's all of a sudden? Tried to upgrade and manual fresh install, but still can't see it? I've tried to change directory where they are located but still doesnt appear. Haven't changed anything else, and seems to have stopped being recognised in the models/VAE folder. Any ideas?

**Comment by brknsoul** [comment]
If you're using an SDXL model, then SD15 VAEs and Loras are hidden from view, and vice-versa.

**Reply by thestorytree** [reply]
no... all my models are 1.5. is there a setting where i can change this option?

**Reply by thestorytree** [reply]
nevermind, fixed it

**Reply by BlackWyvern** [reply]
Care to elaborate? I'm having the same issue.
